ReportService/Web.config

```diff
@@ -9,6 +9,7 @@
     <add key="webpages:Enabled" value="false" />
     <add key="ClientValidationEnabled" value="true" />
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+    <add key="ReportServer" value="http://testreports.caes.ucdavis.edu/reportserver"/>
```

Comment:
@jSylvestre we should replace the server and token, though it might be a manual thing depending on how we choose to deploy